src/main/java/rx/internal/operators/OperatorScan.java

```diff
@@ -270,23 +235,51 @@ public void onCompleted() {
             emit();
         }
         
+        @Override
+        public void request(long n) {
```

Comment:
Okay thanks for your consideration. As I said it will help a lot for me to be able to see changes interleaved.